addons/mail/models/mail_channel.py

```diff
@@ -385,6 +385,9 @@ def message_post(self, *, message_type='notification', **kwargs):
             })
         return message
 
+    def _message_post_after_hook(self, message, msg_vals):
+        self._set_last_seen_message(message)
```

Comment:
Maybe this shouldn't be done for mailing channels? Do we have a notion of new message in those? The UI does not seem to update very well.

Reply:
What do you mean for the UI ? I tested and didn't see anything weird but I'm not sure I'm testing it entirely ?

Reply:
Maybe you fixed it since the comment, I'll just test/review again I guess.